### ISSUE TITLE
Model without mesh error

### ISSUE THREAD

**Issue by ddliugit** [stack-of-tasks/pinocchio#912]
I want to do a small experiment for a simple Urdf file without mesh etc.

see this Urdf file with only one bar:

```
> 
> <?xml version="1.0"?>
> <robot name="model">
>    <link name="base_link"/>
>    <joint name="base_joint" type="fixed">
>     <parent link="base_link"/>
>     <child link="link1"/>
>     <origin rpy="0 0 0" xyz="0 0 0"/>
>   </joint>
>    <link name="link1">
>     <collision>
>       <origin rpy="0 0 0" xyz="0 0 1.0"/>
>       <geometry>
>         <box size="0.1 0.1 2"/>
>       </geometry>
>     </collision>
>     <visual>
>       <origin rpy="0 0 0" xyz="0 0 1.0"/>
>       <geometry>
>         <box size="0.1 0.1 2"/>
>       </geometry>
>       <material name="orange"/>
>     </visual>
>     <inertial>
>       <origin rpy="0 0 0" xyz="0 0 1.0"/>
>       <mass value="1"/>
>       <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0"/>
>     </inertial>
>   </link>
> </robot>
> 
```

**However, Gepetto gui doesn't show the box .**
BOX does not exist.: unspecified iostream_category error
Node "world/pinocchio/collisions/link1_0" does not exist.
BOX does not exist.: unspecified iostream_category error
Node "world/pinocchio/visuals/link1_0" does not exist.

In python I tried it like this way:
```
`# Import model
robot = pin.RobotWrapper.BuildFromURDF(
    urdf_model_path, [mesh_dir], pin.JointModelFreeFlyer())  # floating base
model = robot.model
data = model.createData()

robot.initDisplay(loadModel=True)
robot.display(robot.q0)`
```

**Can you help me out?**

**Comment by gabrielebndn** [comment]
May be related to #726. In short, right now Pinocchio's routines are only able to show meshes. This should be fixed by enhancing GepettoVisualizer https://github.com/stack-of-tasks/pinocchio/blob/master/bindings/python/pinocchio/visualize/gepetto_visualizer.py

**Comment by gabrielebndn** [comment]
So, this requires python bindings for [hpp-fcl](https://github.com/humanoid-path-planner/hpp-fcl).
I was about to make them myself inside Pinocchio, but @jmirabel told me that they are being made inside hpp-fcl itself: they are basically ready on the [refactoring](https://github.com/humanoid-path-planner/hpp-fcl/tree/refactoring) branch, which should be merged in a matter of weeks, after the necessary compatibility checks.
This being the case, it would be wasteful and counter-productive to develop the bindings in Pinocchio. Therefore, we will wait for a new release of hpp-fcl.

**Comment by ddliugit** [comment]
OK, I see. Thanks.

**Comment by gabrielebndn** [comment]
@ddliugit, I think we can keep this issue open until it is solved

**Comment by ddliugit** [comment]
OK, that's great.

**Comment by gabrielebndn** [comment]
Solved by #1004
